custom_components/button_plus/__init__.py

```diff
@@ -7,10 +7,11 @@
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers.device_registry import DeviceEntry
 
-from .button_plus_api.model import DeviceConfiguration
-from .buttonplushub import ButtonPlusHub
+from custom_components.button_plus.button_plus_api.model import DeviceConfiguration
+from custom_components.button_plus.buttonplushub import ButtonPlusHub
+# from custom_components.button_plus.coordinator import ButtonPlusCoordinator
```

Comment:
If we are not going to use the Coordinator here then lets just remove it instead of commenting it out.

Reply:
Changed this back